src/v0/destinations/pinterest_tag/utils.js

```diff
@@ -38,23 +88,11 @@ const USER_NON_ARRAY_PROPERTIES = ['client_user_agent', 'client_ip_address'];
  * Ref: https://s.pinimg.com/ct/docs/conversions_api/dist/v3.html
  */
 const processUserPayload = (userPayload) => {
-  let formatValue = '';
   Object.keys(userPayload).forEach((key) => {
     switch (key) {
```

Comment:
Remove this repeating switch case here & handle default `String(userPayload[key])` in the getHashedValue function

Reply:
This can just be
Object.keys(userPayload).forEach((key) => {
    userPayload[key] = getHashedValue(key, userPayload[key]);
}

Reply:
Updated